src/Components/CreateImageWizardV2/steps/Oscap/OscapProfileInformation.tsx

```diff
@@ -74,6 +82,55 @@ const OscapProfileInformation = (): JSX.Element => {
             <TextListItem component={TextListItemVariants.dd}>
               {oscapProfileInfo.openscap?.profile_id}
             </TextListItem>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Kernel arguments:
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              <CodeBlock>
+                <CodeBlockCode id="code-content">
+                  {oscapProfileInfo?.kernel?.append}
+                </CodeBlockCode>
+              </CodeBlock>
+            </TextListItem>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Disabled services:
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              <CodeBlock>
+                <CodeBlockCode id="code-content">
+                  {disableServicesDisplayString}
+                </CodeBlockCode>
+              </CodeBlock>
+            </TextListItem>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Enabled services:
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              <CodeBlock>
+                <CodeBlockCode id="code-content">
+                  {enabledServicesDisplayString}
+                </CodeBlockCode>
+              </CodeBlock>
+            </TextListItem>
+            <Alert
+              variant="info"
+              isInline
+              isPlain
+              title="Additional customizations"
+            >
+              Selecting an OpenSCAP profile will cause the appropriate packages,
+              file system configuration, kernel arguments, and services to be
+              added to your image.
+            </Alert>
```

Comment:
Alert will need to be put outside of the TextList, see https://github.com/osbuild/image-builder-frontend/pull/1611#issuecomment-1933752350